parts/k8s/kubernetesjumpboxcustomdata.yml

```diff
@@ -0,0 +1,15 @@
+#cloud-config
+
+write_files:
+
+- path: "/home/{{WrapAsVariable "jumpboxUsername"}}/.kube/config"

```

Comment:
Is there a possible race condition between cloud-init and user `<jumpBoxUsername>` creation on the host OS?

Reply:
I don't think so jumpboxUsername is a variable just like we use variables in master custom data, it stays as a variable up until deployment and jumpboxUsername is passed in from the apimodel

Reply:
Sorry if I wasn't clear: `/home/<jumpboxUsername>/` won't exist until the user `<jumpboxUsername>` is created. What process creates that user, and does it happen before cloud-init?

Reply:
I checked and it's not the case